utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -68,54 +71,54 @@ pub struct StorageParams {
 	pub hostinfo: HostInfoParams,
 
 	/// Skip the `read` benchmark.
-	#[clap(long)]
+	#[arg(long)]
 	pub skip_read: bool,
 
 	/// Skip the `write` benchmark.
-	#[clap(long)]
+	#[arg(long)]
 	pub skip_write: bool,
 
 	/// Specify the Handlebars template to use for outputting benchmark results.
-	#[clap(long)]
+	#[arg(long)]
 	pub template_path: Option<PathBuf>,
 
 	/// Add a header to the generated weight output file.
 	///
 	/// Good for adding LICENSE headers.
-	#[clap(long, value_name = "PATH")]
+	#[arg(long, value_name = "PATH")]
 	pub header: Option<PathBuf>,
 
 	/// Path to write the raw 'read' results in JSON format to. Can be a file or directory.
-	#[clap(long)]
+	#[arg(long)]
 	pub json_read_path: Option<PathBuf>,
 
 	/// Path to write the raw 'write' results in JSON format to. Can be a file or directory.
-	#[clap(long)]
+	#[arg(long)]
 	pub json_write_path: Option<PathBuf>,
 
 	/// Rounds of warmups before measuring.
-	#[clap(long, default_value = "1")]
+	#[arg(long, default_value = "1")]
 	pub warmups: u32,
 
 	/// The `StateVersion` to use. Substrate `--dev` should use `V1` and Polkadot `V0`.
 	/// Selecting the wrong version can corrupt the DB.
-	#[clap(long, possible_values = ["0", "1"])]
+	#[arg(long, value_parser = PossibleValuesParser::new(["0", "1"]).map(|v| v.parse::<u8>().expect("0 and 1 can be parsed to u8; qed")))]
```

Comment:
```suggestion
	#[arg(long, value_parser = clap::value_parser!(u8).range(0..=1))]
```

Shouldn't that work?